### PR TITLE
Function samples - initial release

### PR DESCRIPTION
**Pull request by marekq**

### CODE REVIEW

**Comment by marekq** [comment]
I still need to add screenshots and get the text reviewed, but curious to start gathering feedback.

File: stedi-cloud/functions/hello-world/readme.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+  console.log("EVENT: " + JSON.stringify(event, null, 2));
+  return {
+    logStreamName: context.logStreamName
+  };
+};
+```
+
+## Testing the sample Function
+
+To get a feeling for how functions behave, we encourage to simply test this sample Function by pressing "Execute" on the top right of the screen. The Function will invoke and after a few seconds, you should see the output results of the execution and what was submitted to the Function logs.
```
Comment (MattPDengler) [inline stedi-cloud/functions/hello-world/readme.md:39]: Change first `functions` to `Functions`

File: stedi-cloud/functions/hello-world/readme.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+- *Duration*: The exact duration of the request.
+- *Billed Duration*: The billed duration of the request
+- *Memory size*: Memory your Function had available, default on 1024MB
+- *Memory used*: Memory your function consumed during execution.
+- *Init duration*: Time it took to prepare and start the function (also known as the "cold start").
+
+<screenshot-execute-hello-world>
+
+## Writing our own function
```
Comment (MattPDengler) [inline stedi-cloud/functions/hello-world/readme.md:64]: -> `Function`

File: stedi-cloud/functions/hello-world/readme.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+
+`return {
+    logStreamName: context.logStreamName
+  };`
+
+For the rest, it's up to you to decide what your function does in code.
+
+### Create a new Function
+
+Now that we went over some rules, go ahead and create a new Stedi Function in Terminal, which you can call "GreeterFunction". In this Function, we take the input from the event handler, add a simple string and return it back to the client. The function is only a few lines long;
```
Comment (MattPDengler) [inline stedi-cloud/functions/hello-world/readme.md:95]: last `function` -> `Function`

File: stedi-cloud/functions/web-request/readme.md [hunk 4]
```diff
@@ -0,0 +1,244 @@
...
+      "created_at": "2022-04-13T22:33:56.220Z",
+      "function_execution_id": "fc063c1f-4141-4ec9-a4cf-412e6d6956ed"
+    }
+  ]
+}
+```
+
+### Conclusion
+
+This concludes our webrequest tutorial on Functions. We hope you enjoyed the experience and will build great solutions with Stedi Cloud!
```
Comment (MattPDengler) [inline stedi-cloud/functions/web-request/readme.md:244]:
```suggestion
This concludes our webrequest tutorial on Functions. We hope you enjoyed the experience and will build great solutions with Stedi Cloud! For questions, reach out to us [here](https://www.stedi.com/contact). 
```

**Review by kasiafojucik — APPROVED** [review]

**Pull request closed without merging**